content/en/logs/guide/aws-account-level-logs.md

```diff
@@ -0,0 +1,266 @@
+14. Review the stack options, and click the checkbox stating `I acknowledge that AWS CloudFormation might create IAM resources with custom names`.
+15. Click **Submit**.
+
+### Manual
+
+{{< tabs >}}
+{{% tab "Lambda Forwarder" %}}
+
+1. If you haven't already, set up the [Datadog Forwarder][101] Lambda function.
+2. Use the [AWS CLI][102] to grant CloudWatch Logs the permission to execute your function.
+   - Replace `<REGION>` with the region containing your Datadog Forwarder Lambda function.
+   - Replace `<ACCOUNT_ID>` with your 12-digit AWS account ID (excluding dashes).
+
+```bash
+aws lambda add-permission \
+  --region "<REGION>" \
+	--function-name "forwarder-function" \
+	--statement-id "forwarder-function" \
+	--principal "logs.amazonaws.com" \
+	--action "lambda:InvokeFunction" \
+	--source-arn "arn:aws:logs:<REGION>:<ACCOUNT_ID>:log-group:*" \
+	--source-account "<ACCOUNT_ID>"
+```
+
+3. Create an account-level subscription filter policy. In the example provided below, all log events that contain the string `ERROR` are streamed, except those in the log groups named `LogGroupToExclude1` and `LogGroupToExclude2`.
+   - Replace `FORWARDER_ARN` with the ARN of the Datadog Forwarder Lambda function.
+
+```bash
+aws logs put-account-policy \
+  --policy-name "ExamplePolicyLambda" \
+  --policy-type "SUBSCRIPTION_FILTER_POLICY" \
+  --policy-document '{"DestinationArn":"<FORWARDER_ARN>", "FilterPattern": "", "Distribution": "Random"}' \
+  --scope "ALL"
+```
+
+**Note**: To exclude certain log groups from log forwarding, use the `--selection-criteria` option as outlined in the [command reference][103].
+
+[101]: /logs/guide/send-aws-services-logs-with-the-datadog-lambda-function/
+[102]: https://aws.amazon.com/cli/
+[103]: https://docs.aws.amazon.com/cli/latest/reference/logs/put-account-policy.html
+{{% /tab %}}
+{{% tab "Amazon Data Firehose" %}}
+
+#### Create S3 bucket and role for Amazon Data Firehose
+
+The following steps guide you through creating a bucket and IAM role. This role grants Amazon Data Firehose permission to put data into your Amazon S3 bucket in case of delivery failures.
+
+1. Use the [AWS CLI][201] to create an S3 Bucket. Optionally, you can use an existing bucket.
+   - Replace `<BUCKET_NAME>` with the name for your S3 bucket.
+   - Replace `<REGION>` with the region for your S3 bucket.
+
+```
+aws s3api create-bucket \
+  --bucket MY-BUCKET \
+  --create-bucket-configuration LocationConstraint=<REGION>
+```
+
+2. Create a `TrustPolicyForFirehose.json` file with the following statement:
+
+```bash
+{
+  "Statement": {
+    "Effect": "Allow",
+    "Principal": { "Service": "firehose.amazonaws.com" },
+    "Action": "sts:AssumeRole"
+    } 
+}
+```
+
+3. Create an IAM role, specifying the trust policy file:
+
+```bash
+aws iam create-role \
+  --role-name FirehosetoS3Role \
+  --assume-role-policy-document file://./TrustPolicyForFirehose.json
+```
+   **Note**: The returned **Role.Arn** value is used in a later step.
```

Comment:
Recommend moving up:
Create an IAM role, specifying the trust policy file. **Note**: The returned **Role.Arn** value is used in a later step.